init.lua

```diff
@@ -380,24 +408,25 @@ local function lootCorpse(corpseID)
                 local stackable = corpseItem.Stackable()
                 local freeStack = corpseItem.FreeStack()
                 if corpseItem.Lore() then
-                    local haveItem = mq.TLO.FindItem(('=%s'):format(corpseItem.Name()))()
-                    local haveItemBank = mq.TLO.FindItemBank(('=%s'):format(corpseItem.Name()))()
-                    if haveItem or haveItemBank or freeSpace <= loot.SaveBagSlots then
+                    local haveItem = mq.TLO.FindItem(('=%s'):format(corpseItem.Name()))() or -1
+                    local haveItemBank = mq.TLO.FindItemBank(('=%s'):format(corpseItem.Name()))() or -1
+                    if haveItem~= -1 or haveItemBank~= -1 or freeSpace <= loot.SaveBagSlots then
```

Comment:
what was the purpose of the -1s?
`if <nullable field> then` will be false when the field is `nil`, which FindItem returns if the item is not found.

Reply:
`nil` is not a string, `NULL` is a string, but we are evaluating the FindItem to a lua primitive, not leaving it as userdata, so these `nil` are not strings, they are actual lua `nil`.

Reply:
you're right. I don't know what i was thinking.